tools/probes/common/check_voms

```diff
@@ -11,13 +11,13 @@
 # - Cedric Serfon, <cedric.serfon@cern.ch>, 2014-2016
 # - David Cameron, <david.cameron@cern.ch>, 2015
 
-import commands
 import os
 import re
 import sys
 import time
 import types
 
+import ldap
```

Comment:
this has to be added to the requirements.txt?

Reply:
Since it is only a probe, I wouldn't add it.